### ISSUE TITLE
Install octopus CLI action fails on a self-hosted windows runner

### ISSUE THREAD

**Issue by stambunan** [OctopusDeploy/install-octopus-cli-action#223]
**Describe the bug**
Related to https://github.com/OctopusDeploy/install-octopus-cli-action/issues/139.

I'm using the latest Octopus step (1.1.8) and I'm still getting the error:

`Expand-Archive : is not a supported archive file format. .zip is the only supported archive file format.` 

**To Reproduce**
Use the following:
```
      - name: Install Octopus CLI
        uses: OctopusDeploy/install-octopus-cli-action@v1.1.8
        with:
          version: latest
```
**Expected behavior**
Installation of Octopus CLI should be successful

**Screenshots**

<img width="1580" alt="image" src="https://user-images.githubusercontent.com/29392958/151725473-256c0699-88b3-4acf-8e00-a7951614b61c.png">

Powershell Version: 
-------
5.1.17763.2268

**Comment by johnsimons** [comment]
@stambunan I noticed the fix for this is not in v1.1.8, we will be releasing a new version very shortly but in the meantime, can you try `uses: OctopusDeploy/install-octopus-cli-action@main` ?

**Comment by stambunan** [comment]
@johnsimons Looks like it's still showing the same error
![image](https://user-images.githubusercontent.com/29392958/151726912-1339de0d-a94b-451f-a20e-78783ec77215.png)

**Comment by johnsimons** [comment]
We have just published v1.1.9

**Comment by johnsimons** [comment]
That is really weird @stambunan, I am not sure what is going on.
Are you using a self hosted runner?

**Comment by johnsimons** [comment]
I have just noticed the title of the issue 🤦

**Comment by johnsimons** [comment]
We are testing this action using the latest windows, and it seems it uses PowerShell Core.
See https://github.com/OctopusDeploy/install-octopus-cli-action/runs/5000096958?check_suite_focus=true
![image](https://user-images.githubusercontent.com/122651/151727884-7de281d6-c8a8-4113-b7dc-939c3a4162c5.png)
That seems to be the only difference.

**Comment by johnsimons** [comment]
What Windows version are you running?

**Comment by stambunan** [comment]
Ah that might be why. We're using server 2019. I've managed to work around this by installing octo.exe manually and adding that to the system path. I've then removed this step and was able to use the create release action just fine.

**Comment by johnsimons** [comment]
I thought this would have worked on non PS Core, when I look at the [code in Powershell](https://github.com/actions/toolkit/blob/daf8bb00606d37ee2431d9b1596b88513dcf9c59/packages/tool-cache/src/tool-cache.ts#L353), which is part of the github action lib, it attempts to call .net code, do you have .NetFramework installed in this machine?

**Comment by hnrkndrssn** [comment]
Hey @stambunan if you add the Install CLI action as below (using the version which has the fix in it) does it work for you on your self-hosted Windows runner?

```
- name: Install Octopus CLI
        uses: OctopusDeploy/install-octopus-cli-action@v1.1.9
        with:
          version: latest
```

I'm not sure why when you tried with `OctopusDeploy/install-octopus-cli-action@main` didn't work as it **should** use what the latest code on `main` branch is but looking at the log output it didn't seem to be as it was still missing the `.zip` extension on the path.

**Comment by stambunan** [comment]
Hey @hnrkndrssn , we've just installed octopus CLI on the server and that seems to work. So we didn't need to add a step that installs the cli. We might use this step in the future but for now, I think we're happy to keep it as is. 

@johnsimons , yep we have .NetFramework and Core installed in the build machine. We use it to build our Framework and Core projects.

Happy to close this for now unless there are any other concerns as I think the new version of CLI action should resolve this? If not, as a workaround, the octo.exe can be installed to the build server.

**Comment by johnsimons** [comment]
@stambunan  Thanks for confirming the fix.
Both options should work either install cli directly on the agent or use the step.

Any other issues let us know.

**Comment by justin-newman** [comment]
I am also running into this error with version 1.1.9 of the cli installer on my self-hosted Windows Server 2019 machine. I have latest .net install as well as some older versions back to full framework.

![image](https://user-images.githubusercontent.com/23383538/157966063-80ba5426-1e92-4433-93a7-676c32a8b4cf.png)

**Comment by johnsimons** [comment]
Hi @justin-newman, 

Sorry to hear that you also have issues with this.
Can you please [enable debug](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) logging and send us the logs?

I want to see what [this line](https://github.com/OctopusDeploy/install-octopus-cli-action/blob/e3ed9e97216cc85b53a82598e53502def7402814/src/octopus-cli.ts#L72) looks like.

**Comment by johnsimons** [comment]
@stambunan and @justin-newman,

I finally got to the bottom of this issue, so in the end, my fix in [this PR](https://github.com/OctopusDeploy/install-octopus-cli-action/pull/216) was supposed to have gone out in v1.1.9 but it never did!
So I have just released v1.1.10 which now contains the fix to this issue, so please update your workflow to use this new version.

Sorry for the confusion.

**Comment by justin-newman** [comment]
Thank you very much. I'm sorry I didn't get back to you in time, I went on vacation basically right after commenting and then ofc I just forgot all about it.
